opencsp/app/sofast/lib/DistanceOpticScreen.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass, field
+
+from opencsp.common.lib.geometry.Vxyz import Vxyz
+import opencsp.common.lib.tool.hdf5_tools as h5
+
+
+@dataclass
+class DistanceOpticScreen(h5.HDF5_IO_Abstract):
+    """Represents a distance measurement between the optic and the center of the screen. This measurement is typically
+    achieved by displaying the crosshairs from the SofastGUI and measuring from the center of the optic
+    (measurement_point 0,0,0) to the center of the crosshairs."""
```

Comment:
Could we mention that this measurement does not necessarily need to be from the center. Also, for off-axis paraboloids, the center of the optic is not (0, 0, 0). Just trying to reduce potential confusion, but generally I like this.

Reply:
When you say "does not necessarily need to be from the center", you mean the optic, right? The crosshairs at the center of the projection should always be what is used for this measurement?

If so, then I'd like to just double-check that this is better:
```python
"""Represents a distance measurement between the optic and the center of the screen. This measurement is typically
achieved by displaying the crosshairs from the SofastGUI and measuring from the origin of the optic
(measurement_point 0,0,0) to the center of the crosshairs.

The optic's origin is typically the center of the optic for on-axis optics, such as with spherical or flat mirrors
that are symetric around their midpoint."""
```

Reply:
Yes, I did mean the optic. And I'm not sure what is best, I might be worrying about this too much. All of our examples typically use (0, 0, 0) because it is easy to find without measuring anything, but the measure point can be any point that is inside the optic's perimeter. Perhaps something like: 

```
"""Represents a distance measurement between the optic and the origin of the screen coordinate system.
This measurement is typically achieved by displaying the crosshairs from the SofastGUI and measuring
from a point on the optic (measurement_point x,y,z) to the center of the crosshairs.

The measure point is typically the center of the optic for on-axis optics, such as with spherical or flat
mirrors that are symmetric around their midpoint."""
```

Reply:
Updated with your suggestion. I trust that you know this better than I do.

There are some things about how this system works that I still don't understand (big surprise, I know). Maybe some day...